woocommerce/woocommerce-admin
# Onboarding: Add create homepage logic to "Customize Appearance" step

Fixes #2481

Adds the "Create homepage" logic and template.

As discussed on Slack, the return to task list link should be displayed in a notice, however, no scripts from wc-admin are currently run on the edit post page.

### Screenshots

<img width="712" alt="Screen Shot 2019-09-05 at 9 12 51 PM" src="https://user-images.githubusercontent.com/10561050/64345059-38af7880-d022-11e9-8e56-53cb3bb8f5e8.png">

### Detailed test instructions:

1. Go to the "Customize Appearance" step in the task list.
2. Click "Create homepage."
3. Publish the created homepage.
4. Navigate to any other page and note that you're redirected to the task list. (In a follow-up this will instead be a link in a notice to return to the task list instead of a redirect).
5. Return to the task list and make sure the homepage step is no longer showing until you delete the onboarding created homepage.
